src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UnconfiguredProjectTasksService.cs

```diff
@@ -64,7 +64,9 @@ public Task PrioritizedProjectLoadedInHost
 
         public Task SolutionLoadedInHost
         {
+#pragma warning disable VSTHRD110 // Observe result of async calls
```

Comment:
This is unrelated to the change, please do in different PR.

Reply:
This change looks unrelated to the purpose of this pr.

But some warnings appeared after I updated the version of CPS packages.
The update was needed in order to use the  new constructor in `ChainedProjectValueDataSourceBase`.

Reply:
Sure. I can add another PR just to make this update.

Reply:
https://github.com/dotnet/project-system/pull/7424